src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BaseWindow.cs

```diff
@@ -54,11 +54,16 @@ public virtual void Finish(bool result)
             RaiseOnClose(result);
         }
 
-        protected void RaiseOnClose(bool result)
+        protected virtual void RaiseOnClose(bool result)
```

Comment:
Only a `PopupWindow` should need to `ClearOnClose` clearing the `OnClose` handler is generally new functionality that `BaseWindow` has never had/needed.

If i had a choice about the matter, I wouldn't have put `ClearOnClose` in `BaseWindow`. Even though the event is public you just can't set an event to null in a subclass so i was forced to define `ClearOnClose` there.